packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx

```diff
@@ -133,7 +134,7 @@ describe('Vertical stacked bar chart rendering', () => {
     },
     undefined,
     undefined,
-    !isTimezoneSet(Timezone.UTC),
+    !(isTimezoneSet(Timezone.UTC) && env === 'TEST'),
```

Comment:
foreach timezone loop needs to be applied for vertical bar chart, vertical stacked bar chart and line chart.

Reply:
Needs to be fixed in the next iteration.